package.json

```diff
@@ -46,9 +46,8 @@
     "@types/mocha": "^2.2.43",
     "@types/node": "^8.0.25",
     "chai": "^4.1.2",
-    "istanbul": "^0.4.5",
-    "mocha": "^6.0.1",
-    "nyc": "^11.2.1",
+    "mocha": "^9.2.1",
```

Comment:
Should we lock the versions we have just changed?

Reply:
Sure we can. I'll update it